lib/sql.js

```diff
@@ -62,9 +62,9 @@ SQL.prototype.execute = function(sql, vars, options, cb) {
   var isGetRequest = query.length < MAX_LENGTH_GET_QUERY;
 
   debug('About to perform a query to %s', self.sql_api_url);
-  
-  var params = { 
-    q: query, 
+
```

Comment:
Please configure your IDE to remove unneeded whitespace (makes diffs harder to read)